File: src/main/java/catwalks/compat/jei/CatwalksPlugin.java
package catwalks.compat.jei;

import mezz.jei.api.*;

@mezz.jei.api.JEIPlugin
public class CatwalksPlugin extends BlankModPlugin {

	@Override
	public void register(IModRegistry registry) {
		IItemRegistry itemRegistry = registry.getItemRegistry();
		IJeiHelpers jeiHelpers = registry.getJeiHelpers();
		IGuiHelper guiHelper = jeiHelpers.getGuiHelper();
		
//		registry.addRecipeCategories(
//			new DecorationRecipeCategory(jeiHelpers.getGuiHelper());
//		);
		
		registry.addRecipeHandlers(
			new DecorationSplitRecipeHandler(guiHelper),
			new DecorationRepairRecipeHandler(guiHelper)
		);
		
	}
	
}

File: src/main/java/catwalks/compat/jei/CatwalksPlugin.kt
package catwalks.compat.jei

import mezz.jei.api.*

@mezz.jei.api.JEIPlugin
class CatwalksPlugin : BlankModPlugin() {

    override fun register(registry: IModRegistry?) {
        val itemRegistry = registry!!.itemRegistry
        val jeiHelpers = registry.jeiHelpers
        val guiHelper = jeiHelpers.guiHelper

        //		registry.addRecipeCategories(
        //			new DecorationRecipeCategory(jeiHelpers.getGuiHelper());
        //		);

        registry.addRecipeHandlers(
                DecorationSplitRecipeHandler(guiHelper),
                DecorationRepairRecipeHandler(guiHelper))

    }

}
